test-poc/framework/src/main/java/org/keycloak/test/framework/server/DistributionKeycloakTestServer.java

```diff
@@ -0,0 +1,20 @@
+package org.keycloak.test.framework.server;
+
+public class DistributionKeycloakTestServer implements KeycloakTestServer {
+
+    @Override
+    public void start(KeycloakTestServerConfig serverConfig) {
+
```

Comment:
Perhaps just throw a runtime exception here, something like `new RuntimeException("Not implemented");`?